app/scripts/templates/pair/supp_allow.mustache

```diff
@@ -0,0 +1,24 @@
+<div id="main-content" class="card pair-auth">
+  <header>
+    <h1 id="fxa-pair-supp-allow-header">
+      {{#unsafeTranslate}}Approve connections <small>to begin syncing</small>{{/unsafeTranslate}}
```

Comment:
The text here is different to the [text below](https://github.com/mozilla/fxa-content-server/pull/6901/files#diff-f6cf8d9ddb30af4131b37fdc7dd65000R4):

```
Approve this connection <small>from this device</small>
```

Reply:
yeap, `wait` vs `allow` text is different 👍